lib/api-create.js

```diff
@@ -18,17 +20,26 @@ module.exports = class ClientCreateAPI extends API {
 
 	async process({ clients: clientCodes } = this.data) {
 
-		const clientsToCreate = await ModelClient.formatForCreate(clientCodes);
-
 		const ClientModel = ModelFetcher.get(); // se tiene que usar el modelo del servicio
-
 		const model = new ClientModel();
 
+		const clients = ClientModel.additionalFields ? await this.getClients(clientCodes) : clientCodes.map(code => ({ code }));
```

Comment:
Para chequear que no solo exista el getter sino que además tenga contenido
```suggestion
		const clients = ClientModel.additionalFields?.length ? await this.getClients(clientCodes) : clientCodes.map(code => ({ code }));
```